src/tracks/InstrumentTrack.cpp

```diff
@@ -975,6 +971,10 @@ InstrumentTrackView::InstrumentTrackView( InstrumentTrack * _it, TrackContainerV
 	connect( _it, SIGNAL( nameChanged() ),
 			m_tlb, SLOT( update() ) );
 
+	connect( ConfigManager::inst(), SIGNAL( valueChanged( QString, QString, QString ) ),
+			this, SLOT( handleConfigChange( QString, QString, QString ) ),
+			Qt::QueuedConnection );
```

Comment:
Just curious: Why is this better than using `Qt::AutoConnection`?

Reply:
I think I copied this from some commented VST code. Will check this tomorrow.